tools/update-tester/src/main.rs

```diff
@@ -34,62 +35,202 @@ mod installer;
-            connection should use. By default this DB will only be used \
-            to spawn the individual test databases; no tests will run against \
-            it.")
-        (@arg CACHE: -c --cache [cache_dir] "Directory in which to look-for/store \
-            old versions of the Toolkit.")
-        (@arg REINSTALL: --reinstall [versions] "comma-separated list of versions to force reinstall")
-        (@arg ROOT_DIR: +required <dir> "Path in which to find the timescaledb-toolkit repo")
-        (@arg PG_CONFIG: +required <pg_config> "Path to pg_config for the DB we are using")
-        (@arg CARGO_PGX: +required <cargo_pgx> "Path to cargo-pgx (must be 0.4 series or newer)")
-        (@arg CARGO_PGX_OLD: +required <cargo_pgx_old> "Path to cargo-pgx 0.2-0.3 series")
+    let matches = Command::new("update-tester")
+        .about("Update tester for toolkit releases")
+        .subcommand_required(true)
+        .arg_required_else_help(true)
+	.subcommand(
+        Command::new("full-update-test-source")
+            .long_flag("full-update-test-source")
+            .about("Run update-test, building toolkit from source unless a local cache is supplied")
+            .arg(
+                Arg::new("HOST")
+                    .short('h')
+                    .long("host")
+                    .takes_value(true)
+            )
+            .arg(
+                Arg::new("PORT")
+                    .short('p')
+                    .long("port")
+                    .takes_value(true)
+            )
+            .arg(
+                Arg::new("USER")
+                    .short('u')
+                    .long("user")
+                    .takes_value(true)
+            )
+            .arg(
+                Arg::new("PASSWORD")
+                    .short('a')
+                    .long("password")
+                    .takes_value(true)
+            )
+            .arg(
+                Arg::new("DB")
+                    .short('d')
+                    .long("database")
+                    .takes_value(true)
+            )
+            .arg(Arg::new("CACHE").short('c').long("cache").takes_value(true))
+            .arg(Arg::new("REINSTALL").long("reinstall").takes_value(true))
+            .arg(Arg::new("ROOT_DIR").takes_value(true))
+            .arg(Arg::new("PG_CONFIG").takes_value(true))
+            .arg(Arg::new("CARGO_PGX").takes_value(true))
+            .arg(Arg::new("CARGO_PGX_OLD").takes_value(true)),
     )
-    .get_matches();
-
-    let connection_config = ConnectionConfig {
-        host: matches.value_of("HOST"),
-        port: matches.value_of("PORT"),
-        user: matches.value_of("USER"),
-        password: matches.value_of("PASSWORD"),
-        database: matches.value_of("DB"),
-    };
+	.subcommand(
+Command::new("create-test-objects").long_flag("create-test-objects").about("Creates test objects using the currently installed version of Toolkit")
```

Comment:
Can we get some line breaks in here? :)

Indentation and similar problems in other places too.  Probably it all needs a rustfmt.